pkg/terraform/install_test.go

```diff
@@ -31,15 +31,14 @@ func TestMixin_UnmarshalInstallStep(t *testing.T) {
 
 	assert.Equal(t, "Install MySQL", step.Description)
 	assert.Equal(t, "TRACE", step.LogLevel)
-	assert.Equal(t, false, step.Input)
 	assert.Equal(t, false, step.DisableVarFile)
 }
 
 func TestMixin_Install(t *testing.T) {
 	defer os.Unsetenv(test.ExpectedCommandEnv)
 	expectedCommand := strings.Join([]string{
 		"terraform init -backend=true -backend-config=key=my.tfstate -reconfigure",
-		"terraform apply -auto-approve -input=false -var myvar=foo",
+		"terraform apply -auto-approve -var myvar=foo",
```

Comment:
I believe we need to keep this flag here. It just shouldn't be configurable because we always need to specify false when automating terraform.

https://learn.hashicorp.com/tutorials/terraform/automate-terraform

Reply:
Maybe so.  I tested via the [azure-keyvault bundle](https://github.com/getporter/terraform-mixin/tree/main/examples/azure-keyvault) and all actions in that bundle worked fine w/o `input=false` -- but maybe that's because all param values are accounted for?  Perhaps the interactive deal pops up if a var value is missing?  I can add it back in as default to all applicable commands.  I'll test a bit more...

Reply:
Ok, I think it is the case that the lack of `input=false` only shows up when a var is missing a value.  Here is the difference between behaviors.

Without any `input` flag:

```
If you ever set or change modules or backend configuration for Terraform,
rerun this command to reinitialize your working directory. If you forget, other
commands will detect it and remind you to do so if necessary.
var.myvar
  Value for 'myvar'

  Enter a value: ╷
│ Error: No value for required variable
│
│   on variables.tf line 1:
│    1: variable "myvar" {
│
│ The root module input variable "myvar" is not set, and has no default
│ value. Use a -var or -var-file command line argument to provide a value for
│ this variable.
╵
Error: error running command /cnab/app/terraform terraform apply -auto-approve: exit status 1
```

With `input=false`:

```
If you ever set or change modules or backend configuration for Terraform,
rerun this command to reinitialize your working directory. If you forget, other
commands will detect it and remind you to do so if necessary.
╷
│ Error: No value for required variable
│
│   on variables.tf line 1:
│    1: variable "myvar" {
│
│ The root module input variable "myvar" is not set, and has no default
│ value. Use a -var or -var-file command line argument to provide a value for
│ this variable.
╵
err: error running command /cnab/app/terraform terraform apply -auto-approve -input=false: exit status 1
```

So, we see that without the flag, terraform does try to get an interactive value -- but there is no lag, it fails immediately and the container exits.

Anyways, I'm up for adding back in the `input=false` for clarity and to avoid that extra bit of logic/output.

Reply:
Yeah, I like the clarity of the output on the second. So people don't think there _should_ be a way to enter it interactively.